### PR TITLE
added test for LLVMDouble with different values for opt_level

### PR DESCRIPTION
**Pull request by ichumuh**
This is just the test for https://github.com/symengine/symengine/pull/1614.

### CODE REVIEW

File: symengine/llvm_double.h [hunk 1]
```diff
@@ -52,13 +52,14 @@ class LLVMVisitor : public BaseVisitor<LLVMVisitor>
 public:
     llvm::Value *apply(const Basic &b);
     void init(const vec_basic &x, const Basic &b,
-              const bool symbolic_cse = false, int opt_level = 2);
+              const bool symbolic_cse = false, unsigned opt_level = 3);
     void init(const vec_basic &x, const Basic &b, const bool symbolic_cse,
-              const std::vector<llvm::Pass *> &passes);
+              const std::vector<llvm::Pass *> &passes, unsigned opt_level = 3);
```
Comment (isuruf) [inline symengine/llvm_double.h]: @ichumuh, could you add a test for this new interface as well? Sorry, I'm busy now that the semester has started.

**Comment by ichumuh** [comment]
> @ichumuh, could you add a test for this new interface as well? Sorry, I'm busy now that the semester has started.

This interface is already being called in the last llvm test, so I just added `opt_level` at the end.
I think the interface is a bit odd though. At least currently, `create_default_passes` has `opt_level` as it's only parameter as well, so this parameter is passed twice and the case where both `opt_level`s don't match is not handled properly. At least such a test passes.

**Comment by isuruf** [comment]
> At least currently, create_default_passes has opt_level as it's only parameter as well, so this parameter is passed twice and the case where both opt_levels don't match is not handled properly. At least such a test passes.

This gives the user most flexibility in changing the behaviour. So I think this is fine.

**Comment by isuruf** [comment]
Thanks for pushing this through

**Pull request merged**